src/vs/workbench/contrib/webview/browser/pre/index.html

```diff
@@ -5,7 +5,7 @@
 	<meta charset="UTF-8">
 
 	<meta http-equiv="Content-Security-Policy"
-		content="default-src 'none'; script-src 'sha256-bQPwjO6bLiyf6v9eDVtAI67LrfonA1w49aFkRXBy4/g=' 'self'; frame-src 'self'; style-src 'unsafe-inline';">
+		content="default-src 'none'; script-src 'sha256-noHVLQsurkONXmA3fcuAmcZ8UPYm/db88mhm9gAXcvk=' 'self'; frame-src 'self'; style-src 'unsafe-inline';">
```

Comment:
@mjbvz is this change expected?

Reply:
Looks good, it matches what I computed for the <script> tag at https://emn178.github.io/online-tools/sha256.html 

<img width="784" alt="image" src="https://github.com/microsoft/vscode/assets/5047891/41087d86-fb1d-49f7-907f-8b1f7a4e6842">